src/redux/reducers/cacheReducer.ts

```diff
@@ -26,19 +26,29 @@ export interface Comment {
     expiresAt?:number,
 }
 
+export interface QuickComment {
```

Comment:
do also keep track to `username` making the comment... I made a draft using @noumantahir but it was also visible when using @demo.com.

for reference you can check `getDraftPost` and `setDraftPost` method in `realm.js` but no need to follow the same logic.

![Screenshot 2022-06-07 at 12 52 01 PM](https://user-images.githubusercontent.com/6298342/172327383-ab36acaf-e07d-4802-a5b0-05de6fb15fe5.png)